README.md

```diff
@@ -1 +1,16 @@
 "# hibernate-cinema" 
+# Project Description 
+This project represents small online cinema which has functions like: 
+Registration, Login, Add Product To Cart, Complete Order, etc. 
+Users can also read a description of their orders. 
+
+# Build Pattern 
+This project uses MVC pattern, it is structured using Dao and Service 
+layers which make it easier to read for other developers.
+- **_DAO_** layer contains basic CRUD operations with the persistence layer
+- **_Service_** layer contains business logic for the application  
+- **_Servlets_**, implement client-server communicating logic   
+
+# DBMS 
+The project closely works with SQL queries and provides options to store data 
+in comfortable tables, each table has its own purpose. 
```

Comment:
it would be great to add instruction how to run your project

Reply:
Done :)